### PR TITLE
Add .sql as an alias for .query, and add integration functions with polars

### PR DESCRIPTION
**Pull request by Mytherin**

### CODE REVIEW

File: tools/pythonpkg/src/include/duckdb_python/pyrelation.hpp [hunk 1]
```diff
@@ -192,6 +192,8 @@ struct DuckDBPyRelation {
 
 	duckdb::pyarrow::Table ToArrowTable(idx_t batch_size);
 
+	py::object ToPolars(idx_t batch_size);
```
Comment (Mause) [inline tools/pythonpkg/src/include/duckdb_python/pyrelation.hpp:195]: Can you create an object subclass for this instead, similar to what I did for the pandas DataFrame type?

**Review by Mause — CHANGES_REQUESTED** [review]

**Comment by Mytherin** [comment]
Closing this for now - will re-open once CI passes on my fork

File: tools/pythonpkg/src/include/duckdb_python/pandas_type.hpp [hunk 2]
```diff
@@ -17,6 +17,18 @@ class DataFrame : public py::object {
...
 
+class PolarsDataFrame : public py::object {
+public:
+	PolarsDataFrame(const py::object &o) : py::object(o, borrowed_t {}) {
+	}
+	using py::object::object;
+
+public:
+	static bool check_(const py::handle &object) {
+		return !object.is_none();
```
Comment (Mause) [inline tools/pythonpkg/src/include/duckdb_python/pandas_type.hpp:28]: This one is my bad - it should import the class and call isinstance with it. Should probably fix the pandas one too

**Review by Mause — CHANGES_REQUESTED** [review]

File: tools/pythonpkg/src/pyconnection.cpp [hunk 3]
```diff
@@ -973,6 +972,33 @@ duckdb::pyarrow::RecordBatchReader DuckDBPyConnection::FetchRecordBatchReader(co
 	}
 	return result->FetchRecordBatchReader(chunk_size);
 }
+
+static bool IsPolarsDataFrame(py::object entry) {
+	auto polars_df_type = pybind11::module_::import("polars").attr("DataFrame");
+	return py::isinstance(entry, polars_df_type);
```
Comment (Mause) [inline tools/pythonpkg/src/pyconnection.cpp:978]: Once you've fixed the check_ method on the class, you can just use that here with the template variant of this function

File: tools/pythonpkg/src/pyduckdb/connection_wrapper.cpp [hunk 4]
```diff
@@ -279,6 +279,11 @@ duckdb::pyarrow::Table PyConnectionWrapper::FetchArrow(idx_t chunk_size, shared_
 	return conn->FetchArrow(chunk_size);
 }
 
+py::object PyConnectionWrapper::FetchPolars(idx_t chunk_size, shared_ptr<DuckDBPyConnection> conn) {
```
Comment (Mause) [inline tools/pythonpkg/src/pyduckdb/connection_wrapper.cpp:282]: This should return PolarsDataFrame too 😉

**Review by Mause — CHANGES_REQUESTED** [review]

**Pull request closed without merging**